admin_guide/manage_users.adoc

```diff
@@ -9,15 +9,21 @@
 
 toc::[]
 
+[[managing-users-adding-users]]
 == Adding Users
 
 Once new users log in to OpenShift, an account is created for that user and the
```

Comment:
I think for translation reasons we prefer "After" instead of "Once." See table 2.9 https://obriend.fedorapeople.org/WritingStyleGuide/